easy-random-core/pom.xml

```diff
@@ -57,10 +57,6 @@
             <groupId>io.github.classgraph</groupId>
             <artifactId>classgraph</artifactId>
         </dependency>
-        <dependency>
-            <groupId>commons-beanutils</groupId>
```

Comment:
there is one more problem with `commons-beanutils`: it requires to use the latest version and enforces version upgrade. This project respects back compatibility, but I still had compilation issues related to generic types. In other words, if this dependency is kept, I strongly recommend to shade it inside of easy-random jar instead of dependency

Reply:
Also it can cause silent problem (no compilation issue) with runtime failure in production (if the code that calls easy-random is not checked in tests).